example/android/app/build.gradle

```diff
@@ -206,6 +206,7 @@ dependencies {
     }
 
     implementation project(':mattermost.networkclient')
+    implementation project(':react-native-fs')
```

Comment:
Autolink not working?

Reply:
@enahum Thanks for catching! I wasn't paying attention to android just yet and didn't realize it was already autolinked. I removed the unnecessary code in `MainApplication.java` and `settings.gradle` and tested locally that it works fine without those.

Reply:
This line should not be needed either

Reply:
@enahum Removed and tested. Thanks!